### DISCUSSION TITLE
How to translate the Nuclei docs?

### DISCUSSION THREAD

**Discussion by lu4nx** [projectdiscovery/nuclei#4067] [Q&A]
I translated a Chinese version of the document half a year ago, but never merged it, and today the PR was shut down. So where should I put the work I did before?

https://github.com/projectdiscovery/nuclei-docs/pull/126

**Comment by ehsandeep** [comment]
@lu4nx I'm migrating existing docs into nuclei project - https://github.com/projectdiscovery/nuclei/issues/4077

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
